### PR TITLE
updating liguros repos

### PR DESCRIPTION
**Pull request by palica**
- adding new releases and deprecating old

Please merge
Thank you

### CODE REVIEW

File: repos.d/gentoo/liguros.yaml [hunk 1]
```diff
@@ -1,40 +1,38 @@
...
     - name:
-        - core-kit
-        - python-kit
-        - python-modules-kit
-        - nokit
+        - liguros-repo
       fetcher: GitFetcher
       parser: GentooGitParser
-      require_md5cache_metadata: true
+      require_md5cache_metadata: false
```
Comment (AMDmi3) [inline repos.d/gentoo/liguros.yaml:19]: Repositories without metadata are not acceptable.

File: repos.d/gentoo/liguros.yaml [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
       url: https://liguros.gitlab.io/
     - desc: GitLab Organization
       url: https://gitlab.com/liguros/
+    - desc: Official LiGurOS ebuild repository
+      url: 'https://gitlab.com/liguros/{source}/-/tree/{branch}'
   packagelinks:
-    - desc: Git directory
-      url: 'https://gitlab.com/liguros/{subrepo}/-/tree/20.7-release/{srcname}'
+    - desc: Git directory of package
+      url: 'https://gitlab.com/liguros/{source}/-/tree/{branch}/{srcname}'
```
Comment (AMDmi3) [inline repos.d/gentoo/liguros.yaml:32]: You can't use `{source}` and `{branch}` here.

File: repos.d/gentoo/liguros.yaml [hunk 3]
```diff
@@ -1,40 +1,38 @@
...
     - desc: View ebuild
-      url: 'https://gitlab.com/liguros/{subrepo}/-/blob/20.7-release/{srcname}/{srcname|basename}-{rawversion}.ebuild'
-  tags: [ all, production, liguros ]
+      url: 'https://gitlab.com/liguros/{source}/-/blob/{branch}/{srcname}/{srcname|basename}-{rawversion}.ebuild'
+  tags: [ all, development, liguros ]
```
Comment (AMDmi3) [inline repos.d/gentoo/liguros.yaml:35]: The tag is useless. It's either production so it's available in repology, or it shouldn't be added to the config.

**Review by AMDmi3 — CHANGES_REQUESTED** [review]

File: repos.d/gentoo/liguros.yaml (hunk 1 above)
Comment (palica) [reply repos.d/gentoo/liguros.yaml:19]: https://github.com/repology/repology-updater/blob/master/repos.d/gentoo/sabayon.yaml#L15-L16
https://github.com/repology/repology-updater/blob/master/repos.d/gentoo/calculate.yaml#L15-L16

both repos have that set to false.

Where is the problem then to disable that for liguros repo? Please explain.

**Comment by palica** [comment]
Requested changes have been commited. Please review/merge. Thank you

File: repos.d/gentoo/liguros.yaml (hunk 1 above)
Comment (AMDmi3) [reply repos.d/gentoo/liguros.yaml:19]: Other repositories are not related to this case. Metadata is required, change which drops it is a pessimization and won't be accepted.

**Comment by AMDmi3** [comment]
Timeout

**Pull request closed without merging**